public/app/plugins/datasource/grafana-azure-monitor-datasource/components/AnalyticsConfig.tsx

```diff
@@ -92,6 +124,13 @@ export class AnalyticsConfig extends PureComponent<Props> {
           onChange={this.onAzureLogAnalyticsSameAsChange}
           {...addtlAttrs}
         />
+        {showSameAsHelpMsg && (
```

Comment:
This was very nice UX with the re-enter dialog.

Reply:
Thank you :)